### PR TITLE
Fix security errors, stray log, fsevents node-gyp

### PR DESCRIPTION
**Pull request by damassi**
- Fixes GH notification
- Removes stray log 
- Updates lockfile to remove `fsevents` reference below 1.2.4 which was leading to a `node-gyp` compilation mess in yarn on `yarn install`; see [here](https://github.com/strongloop/fsevents/issues/229#issuecomment-426537238).

### CODE REVIEW

File: src/lib/middleware/intercom.ts [hunk 1]
```diff
@@ -3,7 +3,6 @@ import * as crypto from "crypto"
 import { INTERCOM_BUYER_APP_SECRET } from "../../config"
 
 export function addIntercomUserHash(req, res, next) {
-  console.log(INTERCOM_BUYER_APP_SECRET)
```
Comment (joeyAghion) [inline src/lib/middleware/intercom.ts]: I was wondering what this was!

File: src/desktop/apps/rss/test/routes.test.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import Articles from "../../../collections/articles.coffee"
 import Backbone from "backbone"
 import sinon from "sinon"
-const request = require("rewire")("superagent")
-const routes = require("rewire")("../routes.js")
+import request from "superagent"
+let routes = require("rewire")("../routes.js")
```
Comment (damassi) [inline src/desktop/apps/rss/test/routes.test.js]: Led to an odd test failure

**Comment by damassi** [comment]
Going to merge this; let me know if there should be any follow up.

File: src/lib/middleware/intercom.ts (hunk 1 above)
Comment (sweir27) [reply src/lib/middleware/intercom.ts]: Ohhh me too!

**Comment by alloy** [comment]
Great, thanks! 🎉

**Pull request merged**